### ISSUE TITLE
Intermediate Algorithm Scripting: Sum All Odd Fibonacci Numbers

### ISSUE THREAD

**Issue by chenjacky131** [freeCodeCamp/freeCodeCamp#17870]
#### Describe your problem and - if possible - how to reproduce it
sumFibs(4000000) should return 4613732.
this is my code
```javascript
function sumFibs(num) {
  let sum = 0;
  function fib(e){// 1,1,2,3,5,8,13,21,34	
    if(e==0||e==1){
    return 1;
    }else{
    return fib(e-1) + fib(e-2);
    }
  }
  for(let i=0;fib(i)<=num;i++){	
    if(fib(i)%2 == 0){
    }else{
    sum+=fib(i);
    }
  } 
  return sum;
}
sumFibs(4000000);
````

#### Add a Link to the page with the problem
[fcclink](https://learn.freecodecamp.org/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/sum-all-odd-fibonacci-numbers)

#### Tell us about your browser and operating system

* Browser Name: chrome
* Browser Version: 67.0.3396.99
* Operating System: windows 7

#### If possible, add a screenshot here
![1](https://user-images.githubusercontent.com/25630185/42677037-f962f626-86ac-11e8-8e0a-605b0f633453.PNG)

**Comment by joker314** [comment]
The provided code takes over half a second to run. I think that is what is causing the issue. I agree the error should be more verbose.

(For me, on Chrome 67 Windows 10) it takes 658μs to run.

What we should show is "sumFib(400..) took too long to run"

**Comment by chenjacky131** [comment]
@joker314  yes.maybe it's because of that. I change the recursion and it works
